newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/index.js

```diff
@@ -261,6 +283,29 @@ const BuildSection = React.forwardRef<Props, BuildSectionInterface>(
       return actions;
     };
 
+    const createProject = async (
+      i18n: I18nType,
+      settings: ProjectCreationSettings
+    ) => {
+      setIsOpeningProject(true);
+
+      try {
+        let projectMetadata;
+
+        projectMetadata = await onCreateBlank({
+          i18n,
+          settings,
+        });
+
+        if (!projectMetadata) return;
+
+        setPreCreationDialogOpen(false);
+        onOpenProjectAfterCreation({ ...projectMetadata });
```

Comment:
`projectMetadata` only should work I guess

Reply:
Weirdly I've tried removing this but there must be a typing issue somewhere and can't make it work without...
Wasted a lot of time trying to fix, but can't get my head around what's going on 🤔 
(for info, I didn't introduce this, this is the current behavior in place, so no risk to introduce a bug)